client/components/social-buttons/google.js

```diff
@@ -88,8 +89,9 @@ class GoogleLoginButton extends Component {
 			redirect_uri: this.props.redirectUri,
 			callback: ( response ) => {
 				if ( response.error ) {
-					this.props.recordTracksEvent( 'calypso_login_social_button_failure', {
+					this.props.recordTracksEvent( 'calypso_social_button_failure', {
```

Comment:
Similar to the comment in https://github.com/Automattic/wp-calypso/pull/65911, should `startingPoint` be included in the name?
```suggestion
					this.props.recordTracksEvent( `calypso_${startingPoint}_social_button_failure`, {
```

Reply:
> Similar to the comment in https://github.com/Automattic/wp-calypso/pull/65911, should `startingPoint` be included in the name, instead of the event prop?

@vindl Is there a functional benefit to distinguishing in the name instead of an event prop?

I think the intent of a prop in this context is because:
* `client/components/social-buttons/google.js` is a generic component, not necessarily tied to a specific view/context.
* Dynamic event names could be problematic, if someone passed in a new `starting_point` value without realizing they needed to register a new event in Tracks.

Plus, I really don't want to rename all of these Tracks events unless there's a _really_ good reason 😁

Reply:
Per huddle, keeping the naming as-is.